lua/compe/float.lua

```diff
@@ -96,20 +96,20 @@ function M.show(contents, opts)
   -- Clean up input: trim empty lines from the end, pad
   contents = vim.lsp.util._trim(contents, opts)
 
-  local float_options = M.get_options(contents, opts)
-
   -- close if nothing to display
-  if not float_options or #contents == 0 then
+  if #contents == 0 then
     return M.close()
   end
 
   local buf = vim.api.nvim_create_buf(false, true)
   vim.api.nvim_buf_set_option(buf, "bufhidden", "wipe")
+  contents = vim.lsp.util.stylize_markdown(buf, contents, opts)
+
+  local float_options = M.get_options(contents, opts)
 
   -- applies the syntax and sets the lines to the buffer
   opts.width = float_options.width or opts.width
```

Comment:
I think it should be removed.

Reply:
Do you mean to remove the `stylize_markdown` function call? That is used to syntax highlight code blocks in the documentation and remove the code fences (triple back ticks) surrounding the code blocks. It also converts the `---` into the long lines which acts as a separator.

Reply:
No. opts.width/opts.height is can be removed (because these wasn't used after this line, I think)

Reply:
Ah, right got it, pushed a commit for that